runtime.js

```diff
@@ -460,10 +460,12 @@
 
       // Pre-initialize at least 20 temporary variables to enable hidden
       // class optimizations for simple generators.
```

Comment:
Since performance was the only reason I originally bothered pre-initializing these fields, and you've demonstrated (with actual data) that it doesn't help, why don't we just replace this code with
```js
for (var tempIndex = 0, tempName;
     hasOwn.call(this, tempName = "t" + tempIndex);
     ++tempIndex) {
  this[tempName] = null;
}
```
That way, on the first run, there would be only one `hasOwn.call`, and no assignments. Does that make sense?